internal/version/version_test.go

```diff
@@ -170,4 +226,15 @@ var _ = t.Describe("Version", func() {
 			Expect(err).To(HaveOccurred())
 		})
 	})
+	t.Describe("test generating string from info", func() {
+		It("should succeed returning a formatted string", func() {
+			infoString := testInfo.String()
+			Expect(infoString).To(Equal(testInfoStr))
+		})
+		It("should succeed returning a JSON formatted string", func() {
```

Comment:
```suggestion
		It("should succeed returning a JSON document", func() {
```

But also, why a different description?

Reply:
Fixed